manifests/deis-workflow-rc.yml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
       - name: deis-workflow
-        image: deis/workflow:latest
+        image: leeroy.bacongobbler.com/deis/workflow:git-c2992c8
```

Comment:
We should leave this at deis/workflow:latest for now--that's the one that is being built from latest master. If you do `make docker-push` it will replace the image field for you in the manifest that is installed with `make kube-create`.